data/card_catalogue/upgradeable-1.json

```diff
@@ -534,7 +297,9 @@
     "upgrade": "Deflect",
     "tier": "green",
     "details": {
-      "rules": "Any damage dealt to you until your next turn is dealt both to you and your attacker",
+      "rules": {
+        "passive": "Anything thrown on you is repelled with equal force",
+        "active": "Any damage dealt to you from the next attack targeting you is dealt both to you and your attacker",
```

Comment:
Since this is a green card, it should have some kind of identity attached to it. Rubber man seems like a funny one for this. You could use it strategically, bouncing things off of them